include/swift/Driver/Driver.h

```diff
@@ -389,8 +389,11 @@ class Driver {
   /// there is an actual conflict.
   /// \param Args The input arguments.
   /// \param Inputs The inputs to the driver.
+  /// \param BatchModeOut An out-parameter flag that indicates whether to
+  /// batch the jobs of the resulting \c Mode::StandardCompile compilation.
```

Comment:
This comment is good because it will help explain why this routine does not return Mode::BatchMode for the batch case. Probably would be good to add a comment somewhere (Driver.h 68?) explaining the difference between BatchMode being set and the output mode, sigh. Amazing how code can seem less clear with a bit of time.